integration-tests/validation/src/testDebug/kotlin/com/android/designcompose/testapp/validation/RenderAllExamples.kt

```diff
@@ -59,6 +61,7 @@ fun performLiveFetch(dcfOutPath: String?) {
         }
 }
 
+@Category(Fetchable::class)
```

Comment:
So this is so that the fetch task just fetches things in here, but not the other tests?

Reply:
Yes, it's a filter tag.